src/Model/Resolver/GenerateCustomerToken.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/module-customer-graph-ql
+ * @link https://github.com/scandipwa/module-customer-graph-ql
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\CustomerGraphQl\Model\Resolver;
+
+use Magento\Framework\Exception\AuthenticationException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthenticationException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Integration\Api\CustomerTokenServiceInterface;
+use Magento\Framework\Event\ManagerInterface as EventManager;
+
+/**
+ * Customers Token resolver, used for GraphQL request processing.
+ */
+class GenerateCustomerToken implements ResolverInterface
+{
+    /**
+     * @var CustomerTokenServiceInterface
+     */
+    private $customerTokenService;
+
+    /**
+     * @var EventManager
+     */
+    private $eventManager;
+
+    /**
+     * @param CustomerTokenServiceInterface $customerTokenService
+     */
+    public function __construct(
+        CustomerTokenServiceInterface $customerTokenService,
+        EventManager $eventManager
+    ) {
+        $this->customerTokenService = $customerTokenService;
+        $this->eventManager = $eventManager;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!isset($args['email']) || empty($args['email'])) {
```

Comment:
graphQL server already has field validation for required fields, it will fail before it reaches these checks and this essentially is dead code